src/gluonnlp/calibration/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
can we add a check for mxnet version using `nlp.utils.check_version` so that mxnet < 1.7 will throw an err msg